src/dlsproto/node/neo/request/GetRange.d

```diff
@@ -209,7 +209,14 @@ public abstract class GetRangeProtocol_v2: IRequestHandler
             this.saved_exception = this.resources.getException();
             this.lzo = this.resources.getLzo();
 
-            this.run();
+            try
+            {
+                this.run();
+            }
+            finally
+            {
+                this.requestFinished();
```

Comment:
Haha, oops, this wasn't called anywhere before?

Reply:
It was called in the swarm 4 version of the request. However, it wasn't in the swarm 5, and it was OK because `fakedls` doesn't use it.

Reply:
(However, it didn't do much even in the swarm 4 version - as you'll see it in dlsnode's PR - I suspect this was the reason for sporadic segfaults on the neo node.